nss-evening-cohort-26/FE-Hackathon-music-streaming-project
# Public private/ym

## Description
- Added the Discover page so we can see public playlists. 
- Updated the playlistDetail card component so that there are conditionals based on the page the user is on and if the logged-in user is viewing their song on their playlist detail or if it's a public playlist from another user. 
- Added more search options when adding a song to a playlist so you can search by song name, artist and genre. 
## Related Issue
#15 #12 

## Motivation and Context
This allows for users to view public playlists 

## How Can This Be Tested?
Go to the discover page on the navbar and view all public playlist. You can also view the playlist detail page of those public playlist. When creating and updating a playlist you should also have the option to toggle between public or private. 
For search you can just go to the add song to playlist page and test out the search bar.

## Screenshots (if appropriate):

## Types of changes

- [ x] New feature (non-breaking change which adds functionality)
